src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+package src.main.java;
+
```

Comment:
It would help readability to have at least a javadoc header for each class! So the reader roughly knows what the class is for